testsuite/tests/tool-debugger/line-breakpoints/test_module_colon_syntax.ml

```diff
@@ -0,0 +1,22 @@
+(* TEST
+ flags += " -g ";
+ debugger_script = "${test_source_directory}/module_colon_input_script";
+ debugger;
+ shared-libraries;
+ setup-ocamlc.byte-build-env;
+ ocamlc.byte;
+ check-ocamlc.byte-output;
+ ocamldebug;
+ check-program-output;
```

Comment:
This tests looks fragile to maintain because it references line numbers in ways that will change if the test header gets updated later on. The solution for this is to use the TEST_BELOW feature which lets you include the test script as a footer instead, making line numbers more robust. Could you use TEST_BELOW for this test? (Just looking at one usage example in the testsuite should clarify how to use it.)